templates/munstrap/static/css/style-munstrap.css

```diff
@@ -35,10 +35,34 @@ ul.groupview, ul.groupview ul {
 	list-style-type: none;
 }
 .munin-icon {
-   background: url(../img/logo-munin.png) left top; 
+   background: url(../img/logo-munin.png) left top;
```

Comment:
A previous.  I prefer layout changes in a seperate patch posted after the functional ones. It is entirely up to the actual maintainers of this project to say what they prefer though.